flytekit/annotated/schedule.py

```diff
@@ -0,0 +1,150 @@
+import datetime
+import re as _re
+
+import croniter as _croniter
+
+from flytekit.models import schedule as _schedule_models
+
+
+class CronSchedule(_schedule_models.Schedule):
```

Comment:
Add a comment that this is duplicate because we don't want to deal with metaclass?